src/Symfony/Component/DependencyInjection/Tests/Fixtures/xml/legacy-services9.xml

```diff
@@ -33,6 +33,6 @@
       <configurator class="%baz_class%" method="configureStatic1"/>
     </service>
     <service id="factory_service" class="Bar" factory-method="getInstance" factory-service="foo.baz"/>
-    <service id="foo_bar" class="%foo_class%" scope="prototype"/>
+    <service id="foo_bar" class="%foo_class%" shared="false" scope="prototype"/>
```

Comment:
this is to test the output of the XmlDumper. As this service is a legacy service, it only sets the scope to prototype (which remains the same). The only thing this PR changed was also setting shared to false in these cases